StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1625ElementDocumentationMustNotBeCopiedAndPasted.cs

```diff
@@ -75,7 +80,40 @@ internal class SA1625ElementDocumentationMustNotBeCopiedAndPasted : DiagnosticAn
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterCompilationStartAction(HandleCompilationStart);
+        }
+
+        private static void HandleCompilationStart(CompilationStartAnalysisContext context)
+        {
+            context.RegisterSyntaxNodeActionHonorExclusions(HandleDocumentationTrivia, SyntaxKind.SingleLineDocumentationCommentTrivia);
+            context.RegisterSyntaxNodeActionHonorExclusions(HandleDocumentationTrivia, SyntaxKind.MultiLineDocumentationCommentTrivia);
+        }
+
+        private static void HandleDocumentationTrivia(SyntaxNodeAnalysisContext context)
+        {
+            DocumentationCommentTriviaSyntax syntax = context.Node as DocumentationCommentTriviaSyntax;
+
+            HashSet<string> documentationTexts = new HashSet<string>();
```

Comment:
:question: Should we disable this rule by default? It looks like it will inevitably be expensive. What's the rate of positive hits on a project like Roslyn or Code Contracts (I'll test this and reply back)?

/cc @vweijsters @AArnott